dev/archery/archery/integration/runner.py

```diff
@@ -560,31 +583,42 @@ def get_static_json_files():
 def run_all_tests(with_cpp=True, with_java=True, with_js=True,
                   with_csharp=True, with_go=True, with_rust=False,
                   with_nanoarrow=False, run_ipc=False, run_flight=False,
-                  run_c_data=False, tempdir=None, **kwargs):
+                  run_c_data=False, tempdir=None, target_languages="",
+                  **kwargs):
     tempdir = tempdir or tempfile.mkdtemp(prefix='arrow-integration-')
+    print(["before", target_languages])
+    target_languages = list(filter(len, target_languages.split(",")))
```

Comment:
Why the `filter` call?

Reply:
Because `"".split(",")` returns `[""]` not `[]`.
(I surprised the Python behavior because `"".split(",")` returns `[]` with Ruby.)

Reply:
Why would you pass `,`? It's ok to make it an error IMHO.

Reply:
Thanks. I'll use it in #44156.